### ISSUE TITLE
[Official] Elo credit card documentation

### ISSUE THREAD

**Issue by luisibanhi** [braintree/credit-card-type#66]
Hello, this issue is just to request this implementation and check if this is all that you needed to push this feature from backlog.

Official Website: https://dev.elo.com.br/documentacao/tabela-de-bins

This website doesn't a translate to english.

**Comment by crookedneighbor** [comment]
That looks like documentation for their API, not documentation around their bin ranges.

I've talked to my colleagues at Braintree, and we've decided to not add support for ELO to this library at this time. However, I've just opened a PR that will allow you to extend the library and add the card brand yourself. I think this will meet your needs.

https://github.com/braintree/credit-card-type/pull/67

ELO bin ranges in particularly are pretty wide-ranging, so you'll not only need to add the card brand, but likely modify the existing cards. Most notably, VISA, which matches for any card number that starts with 4.
